src/analysis/Covisit.js

```diff
@@ -0,0 +1,95 @@
+import * as datasetUtil from "../dataset";
+import * as util from "../util";
+
+const reportType = "covisit";
+
+// number of covisited links to be added to each content report
+const LINK_LIMIT = 6;
+
+// minimum covisits to consider as actual link
+// 5 means : a minimum of 5 unique visits where two contents were viewed
+const MIN_OCC = 5;
+
+// we provide a configurable function
+const analyseNoConf = (minOcc, linkLimit) => (dataset) => {
```

Comment:
`analyseNoConf` doesnt sound configurable :)

Reply:
ahah it was just a temporary function, it meant analyse function without the conf (so, the conf can be passed), but yeah at least confusing ;) will change this anyway